src/plugins/intel_gna/tests/functional/shared_tests_instances/subgraph_tests/permute_concat_concat_permute.cpp

```diff
@@ -24,10 +23,4 @@ INSTANTIATE_TEST_SUITE_P(smoke_permute_concat_concat_permute,
                                             ::testing::Values(CommonTestUtils::DEVICE_GNA)),
                          PermuteConcatConcatPermute::getTestCaseName);
 
-INSTANTIATE_TEST_SUITE_P(smoke_permute_concat_concat_permute,
```

Comment:
why do we remove this tests?

Reply:
I moved all negative cases to positive test  (we support all of them now), and we could not emulate negative case now  without changing the shared test itself